internal/scanner/scanner.go

```diff
@@ -128,49 +125,64 @@ func (s *Scanner) ScanClusterWideResources() error {
 		return err
 	}
 	// create PolicyReport
-	clusterReport := report.NewClusterPolicyReport("cluster-report")
+	clusterReport := report.NewClusterPolicyReport("clusterwide")
 	clusterReport.Summary.Skip = skippedNum
+	// old policy report to be used as cache
+	previousClusterReport, err := s.reportStore.GetClusterPolicyReport("clusterwide")
+	if err != nil {
+		log.Error().Err(err).Msg("error getting ClusterPolicyReport from store")
+	}
 	// Iterate through all auditableResources. Each item contains a list of resources and the policies that would need
 	// to evaluate them.
 	for i := range auditableResources {
-		auditClusterResource(&auditableResources[i], &s.resourcesFetcher, &s.httpClient, &clusterReport)
+		auditClusterResource(&auditableResources[i], s.resourcesFetcher, &s.httpClient, &clusterReport, &previousClusterReport)
 	}
 	log.Info().Msg("scan finished")
-	err = s.reportStore.UpdateClusterPolicyReport(&clusterReport)
+	err = s.reportStore.SaveClusterPolicyReport(&clusterReport)
```

Comment:
same concerns about substituting the in-mem store, instead of providing a new client store.